install.sh

```diff
@@ -1,8 +1,10 @@
 #!/bin/bash
 
+datahome="${XDG_DATA_HOME:-HOME/.local/share}"
```

Comment:
Cool :+1:

Reply:
Well, that's embarrassing - I missed a `$` before home, just recognized this while looking in the description of #144  where I wrote the example correctly…
Another fixing commit for the last PR, sry!